model/cluster.go

```diff
@@ -194,18 +194,20 @@ type EKSClusterModel struct {
 	ClusterRoleId      string
 	NodeInstanceRoleId string
 
-	LogTypes EKSLogTypes `sql:"type:json"`
+	LogTypes SQLStringArray `sql:"type:json"`
```

Comment:
Assigning the same type to different things is a bad practice. I'd rather you implement the common type once and create aliases for the different use cases. (Makes refactoring, e.g. extending/changing just one of the types, a whole lot easier.) Also, I think `JSONStringArray` would better describe the type than `SQLStringArray`.
```go
type JSONStringArray []string
type EKSLogTypes = JSONStringArray
type APIServerAccessPoints = JSONStringArray
```

Reply:
Thanks, fixed, I agree!